### PR TITLE
Add a visitor for interpreter values

### PR DESCRIPTION
**Pull request by turbolent**
Work towards #84

Add a visitor which allows traversing a whole interpreter object tree. This can e.g. be used for data migration.

### CODE REVIEW

File: runtime/ast/import_test.go [hunk 1]
```diff
@@ -86,6 +86,30 @@ func TestAddressLocation_MarshalJSON(t *testing.T) {
 	)
 }
 
+func TestAddressContractLocation_MarshalJSON(t *testing.T) {
```
Comment (janezpodhostnik) [inline runtime/ast/import_test.go:89]: This isn't related to this PR right?

**Review by janezpodhostnik — APPROVED** [review]
LGTM 🚀

File: runtime/ast/import_test.go (hunk 1 above)
Comment (turbolent) [reply runtime/ast/import_test.go:89]: Good catch! No, this test doesn't belong to this PR, I accidentally committed it. I'll remove

File: runtime/ast/import_test.go [hunk 2]
```diff
@@ -86,6 +86,30 @@ func TestAddressLocation_MarshalJSON(t *testing.T) {
...
+        {
+            "Type": "AddressContractLocation",
+            "Address": "0x1",
+            "Name": "A"
+        }
+        `,
+		string(actual),
+	)
+}
+
```
Comment (turbolent) [inline runtime/ast/import_test.go:112]:
```suggestion
```

**Pull request merged**